ESP8266wifi.cpp

```diff
@@ -591,7 +595,7 @@ WifiMessage ESP8266wifi::getIncomingMessage(void) {
 }
 
 // Writes commands (from PROGMEM) to serial output
-void ESP8266wifi::writeCommand(const char* text1 = NULL, const char* text2) {
```

Comment:
Good catch, but this signature doesn't match the header file either... Should be NULL default on second arg instead.